### ISSUE TITLE
Redshift does not handle eclipses

### ISSUE THREAD

**Issue by kheops2713** [jonls/redshift#184]
During eclipses, it becomes darker just like at night time, and the screen should go red.

**Comment by maandree** [comment]
I guess a could add this sometime in the future to [blueshift](https://github.com/maandree/blueshift), but it feels like overkill for redshift. It should not be too much work to just change temperature manually the few times you see a solar eclipse.

**Comment by jonls** [comment]
Next total eclipse is not until March next year so anyone working on a patch should have plenty of time!

**Comment by rubdos** [comment]
+1
Perhaps we could implement some blueshift at the collapse of our solarsystem? http://io9.com/what-the-death-of-the-sun-will-look-like-471796727

**Comment by jonls** [comment]
@rubdos :+1:

**Comment by corecode** [comment]
But does the ambient light really turn more red?  The atmospheric filtering doesn't change during an eclipse.

**Comment by kheops2713** [comment]
Right, however when it's night time (complete darkness), it is simply the fact that there is no light at all (hence atmospheric filtering does not apply) that makes redshift necessary. So I think the only fact that light decreases, even if atmospheric filtering does not apply, makes that having redshift handle it would make sense :-)
